mantidimaging/gui/dialogs/async_task/view.py

```diff
@@ -73,6 +78,17 @@ def set_progress_plot(self, x: list, y: list):
         self.progress_plot.show()
         self.progress_plot.plotItem.plot(x, y)
 
+    def set_progress_residual_plot(self, residual_image: np.ndarray) -> None:
+        if self.residual_image_view is None:
```

Comment:
we can probably cut a line out here by using `self.residual_image_view` instead of assigning to `residual_image_view`:

```python
    if self.residual_image_view is None:
        self.residual_image_view = ImageView()
        self.residual_image_view.setMinimumSize(600, 400)
        self.PlotVerticalLayout.addWidget(self.residual_image_view)
```

Reply:
That was making `mypy` unhappy. Maybe something to come back to.